examples/tutorials/analysis-time/pulsar_analysis.py

```diff
@@ -198,99 +209,176 @@
+
+excess_map_estimator = ExcessMapEstimator(
+    correlation_radius="0.2 deg", energy_edges=[50 * u.GeV, 10 * u.TeV]
+)
+estimator_results = excess_map_estimator.run(dataset=map_datasets[0])
+
+npred_excess = estimator_results.npred_excess
+sqrt_ts = estimator_results.sqrt_ts
+
+fig, (ax1, ax2) = plt.subplots(
+    figsize=(11, 5), ncols=2, subplot_kw={"projection": npred_excess.geom.wcs}
+)
+
+npred_excess.plot(ax=ax1, add_cbar=True)
+ax1.set_title("npred_excess")
+
+sqrt_ts.plot(ax=ax2, add_cbar=True)
+ax2.set_title("sqrt_ts")
 
 
 ######################################################################
-# We can also do a phase-resolved spectrum. In order to do that, there is
-# the class PhaseBackgroundMaker. In a phase-resolved analysis, the
-# background is estimated in the same sky region but in the OFF-phase
-# zone.
+# Note that here we are lacking statistic because we only use one run of CTA.
+#
+# Phase-resolved spectrum
+# -----------------------
+#
+# We can also do a phase-resolved spectrum.
+# In order to do that, we are going to use the `~gammapy.makers.PhaseBackgroundMake` to create a
+# `~gammapy.makers.SpectrumDatasetOnOff` with the ON and OFF taken in the phase space.
+# Note that this maker take the ON and OFF in the same spatial region.
 #
+# Here to create the `~gammapy.datasets.SpectrumDatasetOnOff`, we are going to redo the whole data reduction.
+# However, note that one can use the `to_spectrum_dataset()` method of `~gammapy.datasets.MapDatasetOnOff`
+# (with the `containement_correction` parameter set to True) if such a `~gammapy.datasets.MapDatasetOnOff`
+# has been created as shown above.
 
 e_true = MapAxis.from_energy_bounds(0.003, 10, 100, unit="TeV", name="energy_true")
 e_reco = MapAxis.from_energy_bounds(0.01, 10, 30, unit="TeV", name="energy")
 
 
 geom = RegionGeom.create(region=on_region, axes=[e_reco])
 
-dataset_empty = SpectrumDataset.create(geom=geom, energy_axis_true=e_true)
+spectrum_dataset_empty = SpectrumDataset.create(geom=geom, energy_axis_true=e_true)
 
-dataset_maker = SpectrumDatasetMaker()
+spectrum_dataset_maker = SpectrumDatasetMaker()
 phase_bkg_maker = PhaseBackgroundMaker(
-    on_phase=on_phase_range, off_phase=off_phase_range
+    on_phase=on_phase_range, off_phase=off_phase_range, phase_column_name="PHASE"
 )
-safe_mask_maker = SafeMaskMaker(methods=["aeff-default", "edisp-bias"], bias_percent=20)
 
-datasets = []
+offset_max = 5 * u.deg
+safe_mask_maker = SafeMaskMaker(methods=["offset-max"], offset_max=offset_max)
+
+spectrum_datasets = Datasets()
 
 for obs in obs_list_vela:
-    dataset = dataset_maker.run(dataset_empty, obs)
-    dataset_on_off = phase_bkg_maker.run(dataset, obs)
-    dataset_on_off = safe_mask_maker.run(dataset_on_off, obs)
-    datasets.append(dataset_on_off)
+    spectrum_dataset = spectrum_dataset_maker.run(spectrum_dataset_empty, obs)
+    spectrum_dataset_on_off = phase_bkg_maker.run(spectrum_dataset, obs)
```

Comment:
Same question here. Why is the safe run after the bkg? It was like this before, was there a specific reason?

Reply:
Yes it was like that before. I changed it the other way around.

Reply:
I imagine the order is not critical here? It is important in case of, ex: the `bkg-peak`